examples/sycl/pvc/flash_attention_v2/pvc_flash_attn_mma.hpp

```diff
@@ -42,6 +42,21 @@
 
 namespace cutlass::gemm::collective {
 using namespace cute;
+
+////////////////////////////////////////////////////////////////////////////////////////////////////
+
+template <typename To_type, typename Engine, typename Layout>
+CUTLASS_DEVICE auto convert_type(Tensor<Engine, Layout> const &tensor) {
+    using From_type = typename Engine::value_type;
+    constexpr int numel = decltype(size(tensor))::value;
```

Comment:
I'm not sure if it's appropriate here, but have you tried `select_packing()` from `sm90_mma_tma_gmma_rs_warpspecialized_mixed_input.hpp`? Seems to be doing something a little more involved to pick `numel`.